packages/components/combobox/src/combobox.stories.ts

```diff
@@ -0,0 +1,329 @@
+  'Skeleton',
+  'Spinner',
+  'Switch',
+  'Tabs',
+  'Tag',
+  'Tag list',
+  'Text area',
+  'Text field',
+  'Toggle button',
+  'Toggle group',
+  'Tooltip'
+];
+
+export default {
+  title: 'Form/Combobox',
+  tags: ['draft'],
+  args: {
+    autocomplete: 'both',
+    disabled: false,
+    filterResults: false,
+    label: 'Label',
+    multiple: false,
+    name: 'combobox',
+    placeholder: 'Choose a component',
+    readonly: false,
+    required: false
+  },
+  argTypes: {
+    autocomplete: {
+      control: 'inline-radio',
+      options: ['off', 'inline', 'list', 'both']
+    }
+  },
+  render: ({
+    autocomplete,
+    disabled,
+    filterResults,
+    hint,
+    label,
+    multiple,
+    name,
+    options,
+    placeholder,
+    readonly,
+    required,
+    value
+  }) => {
+    const onClick = (event: Event & { target: HTMLElement }): void => {
+      event.target.closest('sl-form')?.reportValidity();
+    };
+
+    const onUpdate = (): void => {
+      const form = document.querySelector('sl-form')!,
+        pre = form.nextElementSibling as HTMLPreElement;
+
+      pre.textContent = JSON.stringify(form.value, null, 2);
+    };
+
+    return html`
+      <sl-form @sl-update-state=${onUpdate} @sl-update-validity=${onUpdate}>
+        <sl-form-field .hint=${hint} .label=${label}>
+          <sl-combobox
+            ?disabled=${disabled}
+            ?filter-results=${filterResults}
+            ?multiple=${multiple}
+            ?readonly=${readonly}
+            ?required=${required}
+            .autocomplete=${autocomplete}
+            .name=${name}
+            .placeholder=${placeholder}
+            .value=${value}
+          >
+            ${options?.() ?? html`<sl-listbox>${components.map(c => html`<sl-option>${c}</sl-option>`)}</sl-listbox>`}
+          </sl-combobox>
+        </sl-form-field>
+        <sl-button-bar>
+          <sl-button @click=${onClick}>Report validity</sl-button>
+        </sl-button-bar>
+      </sl-form>
+      <pre></pre>
+    `;
+  }
+} satisfies Meta<Props>;
+
+export const Basic: Story = {};
+
+export const Disabled: Story = {
+  args: {
+    disabled: true
+  }
+};
+
+export const Selected: Story = {
+  args: {
+    value: 'Button bar'
+  }
+};
+
+export const FilterResults: Story = {
+  args: {
+    hint: 'The filterResults property is true, which means the list of options will be filtered based on user input.',
+    filterResults: true
+  }
+};
+
+export const Multiple: Story = {
+  args: {
+    hint: 'The multiple property is true, which means you can select more than 1 option at a time. This will render the selected options as tags.',
+    multiple: true,
+    value: ['Button bar', 'Checkbox']
+  }
+};
+
+export const MultipleStacked: Story = {
+  args: {
+    ...Multiple.args,
+    hint: 'When there is not enough space to display all tags, they will be stacked.',
+    value: ['Button bar', 'Card', 'Checkbox', 'Inline message', 'Menu', 'Panel', 'Spinner', 'Switch']
+  }
```

Comment:
maybe make the viewport or at least the container smaller in this story so it will always stack, also on a big monitor.